packages/api/src/schema/api-schema.yaml

```
@@ -1875,7 +2035,43 @@ paths:
               schema:
                 $ref: "#/components/schemas/error"
     get:
-      description: Lists streams
+      tags:
+        - Stream
+      summary: Retrieve streams
+      description:
+        "GET /stream to retrieve all stream objects with the same userId. The
+        response will include both parent stream objects and the historic stream
+        objects with a parentId (stream objects representing a single live
+        stream session). This historic type of \"child-session\" stream object
+        will be deprecated in an upcoming version of the API in favor of the
+        session object.\n\nGET /stream?streamsonly=1 to retrieve all stream
```

Suggestion will mess up the line size, but we can change the remark here as well
```suggestion
        stream session). The usage of these \"child-session\" stream objects
        is deprecated, use the session endpoints instead. To filter this API not
        to return these child sessions add a `?streamsonly=true` query-string\n\n
        GET /stream?streamsonly=1 to retrieve all stream
```

While at it, mind adding the `streamsonly` query-string to the doc here? Like this https://github.com/livepeer/livepeer-com/blob/bfd3354e5402cb4db57f5f70e893b64d6aff2122/packages/api/src/schema/api-schema.yaml#L2291